parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -4582,3 +4597,19 @@ AlterProcedureWithOption AlterProcedureWithOption() :
         <NO> <WARNING> { return AlterProcedureWithOption.NO_WARNING; }
     )
 }
+
+SqlBeginEndCall SqlBeginEndCall() :
+{
+    SqlIdentifier beginLabel = null;
+    SqlIdentifier endLabel = null;
+    final SqlStatementList statements = new SqlStatementList(getPos());
+    final Span s;
+}
+{
+    [ beginLabel = SimpleIdentifier() <COLON> ]
+    <BEGIN> { s = span(); }
```

Comment:
Why is the span() call here rather than above the beginLabel?

Reply:
It should probably be above, I'll update it